src/AsmResolver.DotNet/Cloning/CallbackCloneListener.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+
+namespace AsmResolver.DotNet.Cloning
+{
+    /// <inheritdoc/>
```

Comment:
We probably want to add some docs on what the specific purpose is of this implementation of the `IMemberClonerListener`. i.e., explain that this implementation that maps the `OnClonedMember` to a delegate.